[WIP] Improved Databricks Docs (#3140)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/connect-your-database.md

<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/bq_oauth_as_user.gif" title="Link Button in dbt Cloud Credentials Screen" />

## Connecting to Databricks
You can connect to Databricks by using [dbt-databricks](/connect-your-database#dbt-databricks) for any accounts on dbt 1.0 or later. We do not recommend using the dbt-spark adapter for Databricks as the dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks. 

> TODO: should we remove any mention of dbt-spark on Databricks?

Reviewer: ```suggestion

```

> You can connect to Databricks by using one of two supported adapters: [dbt-databricks](/connect-your-database#dbt-databricks) and [dbt-spark](/connect-your-database#dbt-spark). For accounts on dbt 1.0 or later, we recommend using the dbt-databricks adapter. The dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks. Connecting to Databricks via dbt-spark will be deprecated in the future. 

Reviewer: ```suggestion
You can connect to Databricks by using [dbt-databricks](/connect-your-database#dbt-databricks) for any accounts on dbt 1.0 or later. We do not recommend using the dbt-spark adapter for Databricks as the dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks. 
```

> TODO: should we remove any mention of dbt-spark on Databricks?

Reviewer: Yes - let's remove the mention of using dbt-spark for Databricks. This is not a best practice anyways
